test/utils.py

```diff
@@ -6,6 +6,16 @@
 from nubison_model.utils import temporary_cwd
 
 
+def get_run_id_from_model_uri(model_uri: str) -> str:
```

Comment:
Can you explain the benefits of turning the run_id in line 116 of the test_model into a function?

Reply:
You can see the function is referenced in two places which it was used in just one place.